src/services/completions.ts

```diff
@@ -1037,10 +1060,17 @@ namespace ts.Completions {
                         !moduleSymbol || !isExternalModuleSymbol(moduleSymbol) ? { kind: SymbolOriginInfoKind.SymbolMemberNoExport } : { kind: SymbolOriginInfoKind.SymbolMemberExport, moduleSymbol, isDefaultExport: false };
                 }
                 else if (preferences.includeCompletionsWithInsertText) {
-                    symbols.push(symbol);
+                    addSymbol(symbol);
                 }
             }
             else {
+                addSymbol(symbol);

```

Comment:
This needs to check `preferences.includeCompletionsWithInsertText` since we don't want to offer this completion without that preference being set.